src/query/generated/proto/prompb/types.proto

```diff
@@ -14,10 +14,14 @@ message Sample {
 message TimeSeries {
   repeated Label labels   = 1 [(gogoproto.nullable) = false];
   repeated Sample samples = 2 [(gogoproto.nullable) = false];
+  MetricType type         = 3;
```

Comment:
this makes m3 not compatible with prometheus metric scraper due to difference in definition: https://github.com/prometheus/prometheus/blob/main/prompb/types.proto#L128